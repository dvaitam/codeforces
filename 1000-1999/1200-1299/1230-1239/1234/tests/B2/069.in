13 9
12 7 8 12 19 3 11 2 15 2 20 6 5
