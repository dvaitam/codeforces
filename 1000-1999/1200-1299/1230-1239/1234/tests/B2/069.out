9
5 6 20 15 2 11 3 19 8
